core/src/test/java/io/questdb/griffin/SqlCodeGeneratorTest.java

```diff
@@ -192,20 +192,20 @@ public void testBindVariableInIndexLookup() throws Exception {
     public void testBindVariableInIndexLookupList() throws Exception {
         assertMemoryLeak(() -> {
             compiler.compile("CREATE TABLE 'alcatel_traffic_tmp' (" +
-                    "deviceName SYMBOL capacity 1000 index, " +
-                    "time TIMESTAMP, " +
-                    "slot SYMBOL, " +
-                    "port SYMBOL, " +
-                    "downStream DOUBLE, " +
-                    "upStream DOUBLE" +
-                    ") timestamp(time) partition by DAY",
+                            "deviceName SYMBOL capacity 1000 index, " +
```

Comment:
Too many changes in the file, hard to see if new tests are added.

Reply:
sorry, this is auto-formatting issue. One test was added `testUtf8TableName()`

Reply:
fixed in https://github.com/questdb/questdb/pull/1623/commits/0186f92481bb59cae4fdd01e7ecfff707800cb49